src/main/resources/templates/menubar.html

```diff
@@ -1,13 +1,22 @@
 <div th:fragment="copy">
-<nav class="navbar fixed-top navbar-expand-lg navbar-light border-bottom pt-2 pb-2" th:fragment="copy">
+<nav class="navbar fixed-top navbar-expand-lg navbar-light border-bottom pt-2 pb-2" th:fragment="copy"
+     ng-controller="GeneralJsController">
     <div class="container">
         <a class="navbar-brand align-items-center" th:href="@{/}">
             <img class="img-fluid"
                  alt="UH Groupings Logo"
                  th:srcset="@{images/uh-groupings-logo.png 1x, images/uh-groupings-logo-2x.png 2x, images/uh-groupings-logo.svg}"
                  th:src="@{images/uh-groupings-logo.png}">
-            <span class="sr-only">UH Groupings</span>
-        </a>
+            <span class="sr-only">UH Groupings</span></a>
+        <span class="user-role rounded-circle seafoam-bg p-1 ml-5"
```

Comment:
In my opinion, I think the departmental icon would look best flush against the UHGroupings logo, without ml-5 on large screens and mid-size screens:
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/dbd8c1ba-f8e9-4919-9a9c-ec9e1ee27e5d)
However, the icon is currently floating in the middle with mid-size screens:
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/96fafb23-e782-4f39-bd32-ceac05c37115)
It should be flush on the left of the UHGroupings logo just like on large screens.

As for small screens, it looks fine:
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/bb8479af-f66f-4c4a-aa50-71ad7fe9a610)